modules/installation-user-infra-machines-static-network.adoc

```diff
@@ -0,0 +1,58 @@
+// Module included in the following assemblies:
+//
+// * installing/installing_bare_metal/installing-bare-metal.adoc
+// * installing/installing_bare_metal/installing-restricted-networks-bare-metal.adoc
+// * installing_bare_metal/installing-bare-metal-network-customizations.adoc
+
+[id="installation-user-infra-machines-static-network_{context}"]
+= Configuring static networking
+
+If you install {op-system-first} from an ISO image, you can add kernel arguments
+when you boot that image to configure the node's networking.
+The following table describes and illustrates how to use those kernel arguments.
+
+.Configuring static networking
+[source,adoc]
+|===
+|Description |Examples
+
+a|To configure an IP address, either use DHCP (`ip=dhcp`) or set an individual
+static IP address (`ip=<host_ip>`). Then identify the DNS server IP address (`nameserver=<dns_ip>`) on each node. This example sets: +
+
+* The node's IP address to `10.10.10.2` +
+* The gateway address to `10.10.10.254` +
+* The netmask to `255.255.255.0` +
+* The hostname to `core0.example.com` +
+* The DNS server address to `4.4.4.41`
+
+|`ip=10.10.10.2::10.10.10.254:255.255.255.0:core0.example.com:enp1s0:none nameserver=4.4.4.41` +
+
+|Specify multiple network interfaces by specifying multiple `ip=` entries.
+|`ip=192.168.122.25 ip=192.168.122.26`
```

Comment:
If we are going to give examples for multiple static IP configured interfaces, I think the examples should be complete.  e.g.

```
ip=10.10.10.2::10.10.10.254:255.255.255.0:core0.example.com:enp1s0:none
ip=10.10.10.3::10.10.10.254:255.255.255.0:core0.example.com:enp2s0:none
```

Reply:
Nice. I'll do that. This has been merged, so I'm going to do future work in a new PR.

Reply:
Done.